### PR TITLE
Add pyod

### PR DESCRIPTION
**Pull request by apcamargo**
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pyod/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - combo
+    - joblib
+    - matplotlib
```
Comment (scopatz) [inline recipes/pyod/meta.yaml]: This should be matplotlib-base.

File: recipes/pyod/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    - numba >=0.35
+    - numpy >=1.13
+    - pip
+    - python
+    - scikit_learn >=0.19.1
+    - scipy >=0.19.1
+    - six
+    - suod
```
Comment (scopatz) [inline recipes/pyod/meta.yaml]: Most of these are not needed to build the package and should be removed from host

File: recipes/pyod/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  home: "https://github.com/yzhao062/pyod"
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
+  summary: "A Python Toolkit for Scalable Outlier Detection (Anomaly Detection)"
+  doc_url: "https://pyod.readthedocs.io"
+
+extra:
+  recipe-maintainers:
+    - yzhao062
```
Comment (scopatz) [inline recipes/pyod/meta.yaml]: @yzhao062 are you willing to be a maintainer?

Comment (apcamargo) [reply recipes/pyod/meta.yaml]: Hi @scopatz! @yzhao062 agreed to be a maintainer here: https://github.com/yzhao062/pyod/issues/164

I should have put this in the PR. I'm sorry

Comment (scopatz) [reply recipes/pyod/meta.yaml]: @yzhao062  will have to agree here

**Comment by apcamargo** [comment]
@scopatz I'm having a problem with this recipe because it depends on `combo` and `suod`, which in turn depend on `pyod`. What should be done hore?

**Comment by scopatz** [comment]
If there is truly a circlular dependency you should try to figure out how to build at least one of the packages without the others to get bootstrapped

**Comment by yzhao062** [comment]
Sorry for the confusion. For PyOD, drop SUOD and combo if needed. They are listed for example/demo purpose only.

**Comment by apcamargo** [comment]
Thank you, @yzhao062 

Does combo really depends on pyod? I'll probably write a PR for it too.

Also, do you agree in being added as a maintainer in this recipe (and combo's)?

**Comment by yzhao062** [comment]
@apcamargo Thanks for that. similarly, PyOD is needed for running certain examples in combo, while it is optional for the major models.

One of the reasons I did not create a conda channel is the concern regarding the maintenance. I am happy to be one of the recipe maintainers in case some urgent fix is needed :)

Again, thanks for pushing this :) If you are happy I could add you here: https://pyod.readthedocs.io/en/latest/about.html

**Comment by apcamargo** [comment]
Thank you, @yzhao062! I'd be very happy to be in the list!

@scopatz Can you help me with something? I'm getting these errors in Linux and OSX and I don't know what is causing them:

Linux:
```
conda.CondaMultiError: [Errno 17] File exists: 'libopenblasp-r0.3.6.so' -> '/home/conda/staged-recipes/build_artifacts/pyod_1583347740526/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libblas.so'
()
```

Mac:
```
conda.CondaMultiError: [Errno 17] File exists: 'libopenblasp-r0.3.3.dylib' -> '/usr/local/miniconda/conda-bld/pyod_1583347771045/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/libcblas.dylib'
()
```

**Comment by scopatz** [comment]
Pinging @conda-forge/staged-recipes for help

**Comment by apcamargo** [comment]
@scopatz Seems like there was something wrong with the CI at the time. The recipe builds fine now.

File: recipes/pyod/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 22cd28595187ca54e06418522b699b80ab6eae658ac3de194c3ef85730127016
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (scopatz) [inline recipes/pyod/meta.yaml]: this can be noarch python

Comment (apcamargo) [reply recipes/pyod/meta.yaml]: Done!

**Comment by apcamargo** [comment]
Besides the `CERTIFICATE_VERIFY_FAILED` error (I don't know what is causing it) I think that both this and combo's recipe are ready.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**